### PR TITLE
WIP feat: replace crisp api integration with crisp form

### PR DESCRIPTION
**Pull request by rmonnier9**
- Simplification technique et Correction d'un bug.
- Zones impactées : `FeedbackModal`.
- Détails :
  - Suppression de l'intégration api crisp
  - Utilisation du form crisp

TODO : 

- [x] autofill form with email
- [x] look if possible to not use a public env variable
- [ ] add custom CSS to the form
- [x]  tag to distinguish agent and public

image

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -34,7 +34,5 @@ FRANCECONNECT_CLIENT_SECRET=
 FRANCECONNECT_URL=https://fcp.integ01.dev-franceconnect.fr
 FRANCECONNECT_REDIRECT_URI=http://localhost:3000/api/auth/france-connect/callback
 FRANCECONNECT_POST_LOGOUT_REDIRECT_URI=http://localhost:3000/api/auth/france-connect/logout-callback
-CRISP_TOKEN_ID=
-CRISP_TOKEN_KEY=
-CRISP_WEBSITE_ID="064fca1b-bdd6-4a81-af56-9f38e40953ad"
+NEXT_PUBLIC_CRISP_WEBSITE_ID=064fca1b-bdd6-4a81-af56-9f38e40953ad
```
Comment (XavierJp) [inline .env.dev:37]: why use an env variable if you commit it in the dev ? Or is it a dev SITE_ID ?

**Review by XavierJp — APPROVED** [review]

**Comment by XavierJp** [comment]
Coulod not make it work in local. But sounds great. Do you plan on completing this todo before merging ?

File: .env.dev (hunk 1 above)
Comment (rmonnier9) [reply .env.dev:37]: It is possible to have a different crisp website id in staging/dev and production.

At the moment we don't have, so this website id is the one from production and it is indeed possible to remove the env var.

**Comment by rmonnier9** [comment]
> Coulod not make it work in local. But sounds great. Do you plan on completing this todo before merging ?

Yes, I plan to finish the to do today.

I wonder if we can deploy to staging to test the crisp form ?
(we have to allow domains and localhost:3000 is not accepted)

**Comment by XavierJp** [comment]
We must move from a continuous deployment to an auto deploy to staging followed with a manual deployment to production ?

**Pull request merged**